src/internal_events/tcp.rs

```diff
@@ -80,7 +80,7 @@ pub struct TcpConnectionError {
 
 impl InternalEvent for TcpConnectionError {
     fn emit_logs(&self) {
-        warn!(message = "connection error.", error = %self.error);
+        info!(message = "connection error.", error = %self.error, rate_limit_secs = 10);
```

Comment:
I'm :+1: on the rate limit, but I don't see why it should be downgraded to info.

Reply:
Ok, agree!